File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double x, double y) {
        double R = Math.sqrt(x*x + y*y);
        if (R <= 1) {
            return 10;
        } else if (R > 5 && R <= 10) {
            return 1;
        } else if (R> 1 && R<= 5) {
            return 5;
        } else {
            return 0;
        }
        
    }
}

File: solutions/python/darts/1/darts.py
def score(x, y):
    radius_2 = x*x + y*y
    if radius_2 <= 1:
        return 10
    elif radius_2 <= 25:
        return 5
    elif radius_2 <= 100:
        return 1
    else:
        return 0
